### PR TITLE
object properties

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**

### CODE REVIEW

File: src/property.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	 *
+	 * @param{string} spec.object - Instance on which the property is installed.
+	 * @param{string} spec.name - Name of the property, which value of the same
+	 * name is installed on `my` on the instance.
+	 */
+	var property = object.subclass(function(that, my) {
+		my.initialize = function(spec) {
+			my.super(spec);
+			my.name = spec.name;
+			my.object = spec.object;
```
Comment (BenjaminVanRyseghem) [inline src/property.js]: as I previously said, `my.object` doesn't really gives an idea of what it does.

maybe `my.owner` instead or something similar?

File: src/property.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		};
+
+		that.getter = function() {
```
Comment (BenjaminVanRyseghem) [inline src/property.js]: does it need to be public?

File: src/property.js (hunk 1 above)
Comment (BenjaminVanRyseghem) [inline src/property.js]: addressed

File: src/property.js (hunk 2 above)
Comment (NicolasPetton) [inline src/property.js]: Probably not :)

**Pull request merged**